database/database_handler.py

```diff
@@ -0,0 +1,88 @@
+import os
+import subprocess
+import json
+
+from sqlalchemy import create_engine, Column, String, Boolean, Date, Integer, Float
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.exc import SQLAlchemyError
+
+from utils.general import LOGGER
+
+LOCAL_RUN = False
+
+
+def get_db_access_token(client_id):
```

Comment:
Idea: What if we wrap all these functions inside a DBConfigSQLAlchemy class?
In this way when we create an object we can use the `create_connection` to instantiate the `engine` and `session_maker`. I would also move the `declarative_base()` inside here as Class attribute.